go.mod

```diff
@@ -1,24 +1,58 @@
 module github.com/googleforgames/open-saves
 
-go 1.14
+go 1.17
```

Comment:
Is this necessary or can we split the dependency updates to another PR?

Reply:
Not necessary. I'll remove.